_includes/header.html

```diff
@@ -15,9 +15,20 @@
         </a>
         <div class="dropdown-menu">
           {% for sublink in navlink.submenu %}
+          {% if sublink.divideup %}
+            <div class="dropdown-divider"></div>
+          {% endif %}
           <a class="dropdown-item {% if page.url contains sublink.link %}active{% endif %}" href="{{ navlink.directory}}/{{ sublink.link }}">
             {{ sublink.title }}
           </a>
+          {% for subsublink in sublink.subsubmenu %}
+          <a class="dropdown-item {% if page.url contains subsublink.link %}active{% endif %}" href="{{ navlink.directory}}/{{ subsublink.link }}">
+            <span class="mr-3"></span>&bull; {{ subsublink.title }}
```

Comment:
I would omit or at least reduce the extra space in front. It just does not look good.

Reply:
Do you mean that the subpages are indented?

Reply:
Do you mean that the subpages are indented?

Reply:
```suggestion
            &bull; {{ subsublink.title }}
```